dev/.env.traces.dev

```diff
@@ -0,0 +1,3 @@
+OTEL_TRACING_ENABLED=True
+OTEL_SERVICE_NAME=oncall
+OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317
```

Comment:
alternatively, since these are not sensitive values, we could add these to `docker-compose-developer.yml` (see line 26 in that file)